src/icespeak/settings.py

```diff
@@ -182,6 +182,19 @@ class Keys(BaseModel):
     # TODO: Re-implement TTS with Tiro
     tiro: Literal[None] = Field(default=None)
 
+    def __hash__(self):
+        return hash((self.azure, self.aws, self.google, self.tiro))
+
+    def __eq__(self, other):
+        if isinstance(other, Keys):
+            return (self.azure, self.aws, self.google, self.tiro) == (
+                other.azure,
+                other.aws,
+                other.google,
+                other.tiro,
+            )
+        return False
+
```

Comment:
Where is this used? Are we comparing Keys instances anywhere? Or is it for the `assert_called_once_with(...)` method in the tests?

Reply:
Yep was needed for the assert_called_once_with assertion